layouts/partials/head.html

```diff
@@ -30,12 +30,11 @@
   {{ partialCached "head-css.html" . }}
 
 
-  <!-- Google Analytics -->
-  {{- if and (eq hugo.Environment "production") (or .Site.GoogleAnalytics .Site.Config.Services.GoogleAnalytics.ID) }}
-    <link rel="preconnect" href="https://www.googletagmanager.com" crossorigin />
-    {{ partial "google-analytics.html" . }}
-  {{- end }}
-
+<!-- Google Analytics -->
+{{- if and (eq hugo.Environment "production") .Site.Config.Services.GoogleAnalytics.ID }}
+  <link rel="preconnect" href="https://www.googletagmanager.com" crossorigin />
+  {{ partial "google-analytics.html" . }}
```

Comment:
also need to update this place

Reply:
Yes, the change is completely sufficient. Previously there was only a warning. Now, however, there is an error and the project crashes. The change is therefore necessary.

See here:
 -  https://gohugo.io/methods/site/googleanalytics/

Reply:
i mean in this file https://github.com/imfing/hextra/blob/main/layouts/partials/google-analytics.html

Reply:
To be fair, I only made the change I described. I then tested my private project. As there was no warning and no error, the issue was closed for me.

Reply:
@eskopp as I pointed out above, this PR didn't address all the locations where `Site.GoogleAnalytics` is referenced, thus it will not close the issue for me. 

anyway, I will open a following PR to address this.